algorithm_config.yaml

```diff
@@ -67,7 +67,7 @@ inputs:
         fsspec.core.url_to_fs function when reading granule data files.  See
         https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.core.url_to_fs."
       required: false
-      default: '{"default_cache_type": "all", "default_block_size": 8388608}'
+      default: ""
```

Comment:
why wasn't this also switched to the new mmap?

Reply:
Yep, good question. I removed it from the this spot because it is redundant with the default in the code, and when I change the default here, I have to remember to change it in the code too, and vice versa.

It can lead to a "surprise" because the default here gets passed to the code when the user does not supply a value for this input, which then takes precedence over the default in the code, which can be surprising.

This just eliminates any surprising behavior.